scripts/trim-low-abund.py

```diff
@@ -110,9 +101,9 @@ def get_parser():
                         help='remove k-mers below this abundance',
                         default=DEFAULT_CUTOFF)
 
-    parser.add_argument('--normalize-to', '-Z', type=int,
-                        help='base cutoff on this median k-mer abundance',
-                        default=DEFAULT_NORMALIZE_LIMIT)
+    parser.add_argument('--trim-at-coverage', '-Z', type=int,
```

Comment:
I didn't know this worked, but
```python
parser.add_argument('--trim-at-coverage', '--normalize-to', '-Z', type=int,
```
also works (if there is anyone using the `--normalize-to` flag, it won't break for them)